src/scaffoldUtils.ts

```diff
@@ -75,22 +76,20 @@ export async function createFoldersFromJSON(json: any, jsonpath: vscode.Uri): Pr
 			throw new Error('No workspace folder. Workspace must have at least one folder before Didact scaffolding can begin. Add a folder, restart your workspace, and then try again.');
 		}
 		let rootPath: string | undefined;
-		await vscode.commands.executeCommand('workbench.view.explorer').then ( async () => {
-			await vscode.commands.executeCommand('copyFilePath').then ( async () => {
-				await vscode.env.clipboard.readText().then((copyPath) => {
-					try {
-						if (fs.existsSync(copyPath)) {
-							if (fs.lstatSync(copyPath).isDirectory()) {
-								rootPath = copyPath;
-							} else {
-								rootPath = path.dirname(copyPath);
-							}
-						}
-					} catch (err) {
-						console.log(err);
+		await vscode.commands.executeCommand('workbench.view.explorer');
```

Comment:
This is where the stack trace is hit:
```rejected promise not handled within 1 second: Error: Unknown webview handle:6e70bd07-dc69-4c0d-87a3-79f6e4bcb757
stack trace: Error: Unknown webview handle:6e70bd07-dc69-4c0d-87a3-79f6e4bcb757```

Reply:
And if you add anything above it (even an `await delay(1000);` call ahead of it), the stack trace still hits on that line.